server/src/main/java/com/thoughtworks/go/server/service/AgentConfigService.java

```diff
@@ -364,29 +287,46 @@ public String toString() {
 
     @Deprecated
     public void updateAgentResources(final String uuid, final ResourceConfigs resourceConfigs) {
-        updateAgent(new UpdateResourcesCommand(uuid, resourceConfigs), uuid, Username.ANONYMOUS);
+        Agent agent = agentDao.agentByUuid(uuid);
+        agent.setResources(resourceConfigs);
+        saveOrUpdate(agent, null);
+//        updateAgent(new UpdateResourcesCommand(uuid, resourceConfigs), uuid, Username.ANONYMOUS);
     }
 
     public void updateAgentApprovalStatus(final String uuid, final Boolean isDenied, Username currentUser) {
-        updateAgentWithoutValidations(new UpdateAgentApprovalStatus(uuid, isDenied), currentUser);
+        Agent agent = agentDao.agentByUuid(uuid);
+        agent.setDisabled(isDenied);
+        saveOrUpdate(agent, null);
     }
 
-    public void addAgent(AgentConfig agentConfig, Username currentUser) {
-        updateAgent(new AddAgentCommand(agentConfig), agentConfig.getUuid(), currentUser);
+    public void saveOrUpdate(AgentConfig agentConfig, Username currentUser) {
```

Comment:
`currentUser` isn't used any more. We can probably log the currentUser as it was done in `GoConfigDao` to help keep *some* track to who made the change.

Reply:
There is no user associated with any of the agent changes anymore.